client/jetpack-onboarding/steps/stats.jsx

```diff
@@ -79,7 +79,7 @@ class JetpackOnboardingStatsStep extends React.Component {
 				buttonLabel={ translate( 'Activate stats' ) }
 				e2eType="activate-stats"
 				header={ header }
-				illustration="/calypso/images/illustrations/jetpack-stats.svg"
+				illustration="/calypso/images/illustrations/illustration-stats-jetpack.svg"
```

Comment:
I'd name this one `jetpack-stats.svg` instead (we introduced it for this step, so we can just replace the file).

Reply:
👍